src/patternfly/components/Chips/chip-group.scss

```diff
@@ -0,0 +1,48 @@
+@import "../../patternfly-utilities";
+
+.pf-c-chip-group {
+  --pf-c-chip-group--PaddingTop: calc(#{var(--pf-global--spacer--xs)} / 2);
+  --pf-c-chip-group--PaddingRight: var(--pf-global--spacer--sm);
+  --pf-c-chip-group--PaddingBottom: calc(#{var(--pf-global--spacer--xs)} / 2);
+  --pf-c-chip-group--PaddingLeft: var(--pf-global--spacer--sm);
+  --pf-c-chip-group--BorderRadius: var(--pf-global--BorderRadius--sm);
+  --pf-c-chip-group--MarginRight: var(--pf-global--spacer--sm);
+  --pf-c-chip-group--MarginBottom: var(--pf-global--spacer--sm);
+  --pf-c-chip-group--BackgroundColor: transparent;
+  --pf-c-chip-group--m-toolbar--BackgroundColor: var(--pf-global--BackgroundColor--300);
+  --pf-c-chip-group--m-select-BackgroundColor: transparent;
```

Comment:
`--pf-c-chip-group--m-select-BackgroundColor`,`--pf-c-chip-group--m-select-PaddingBottom`, `--pf-c-chip-group--m-select-Width` aren't being used, do we need them?